main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
+import json
+
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_info = json.load(players_file)
+    for player_name, player_info in players_info.items():
+        race_name = player_info["race"]["name"]
+        if not Race.objects.filter(name=race_name):
+            Race.objects.create(
+                name=race_name,
+                description=player_info["race"]["description"]
+            )
```

Comment:
Better to use that get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create